### ISSUE TITLE
Hosts are reported incorrectly as down

### ISSUE THREAD

**Issue by Ramshield** [Icinga/icingadb#271]
Hi,

2 hosts are incorrectly reported as down (hostalive) by IcingaDB. They are not reported by Icinga as down.
Also the performance data makes them appear as responding to ping.

I tried restarting Icinga2, IcingaDB and a reboot.

![Schermafbeelding 2021-04-13 om 08 36 12](https://user-images.githubusercontent.com/3533440/114508325-82b40380-9c34-11eb-9fb7-8e1279e2ec4d.png)

![Schermafbeelding 2021-04-13 om 08 43 39](https://user-images.githubusercontent.com/3533440/114508349-89db1180-9c34-11eb-9292-d1f9e3f64b87.png)

This is not reported by Icinga however:

![Schermafbeelding 2021-04-13 om 08 45 27](https://user-images.githubusercontent.com/3533440/114508469-ad9e5780-9c34-11eb-89ba-4bdd90413222.png)

And:

![Schermafbeelding 2021-04-13 om 08 45 47](https://user-images.githubusercontent.com/3533440/114508526-c3138180-9c34-11eb-9de1-29a44cfd071b.png)

```
root@icinga:/var/log# dpkg -l | grep icinga2
ii  icinga2                              2.12.3-1.buster               amd64        host and network monitoring system
ii  icinga2-bin                          2.12.3-1.buster               amd64        host and network monitoring system - daemon
ii  icinga2-common                       2.12.3-1.buster               all          host and network monitoring system - common files
ii  icinga2-doc                          2.12.3-1.buster               all          host and network monitoring system - documentation
ii  icinga2-ido-mysql                    2.12.3-1.buster               amd64        host and network monitoring system - MySQL support
root@icinga:/var/log# icingadb --version
Icinga DB version: v1.0.0-rc1
```

Icingaweb module versions:
```
Name | Version
director | 1.8.0
icingadb | 1.0.0-rc1
incubator | 0.6.0
ipl | v0.5.0
monitoring | 2.8.2
reactbundle | 0.8.0
```

Please let me know if this is a bug or if I can correct this somehow. Thank you!

**Comment by lippserd** [comment]
Hi @Ramshield,

We rewrote most of the code for RC2. It would be great if you could give the latest master a try. Please make sure to call `flushall` in Redis and upgrade the database schema before starting Icinga DB again. If this is still a problem then, please open a new issue.

All the best,
Eric
